schema_salad/tests/test_print_oneline.py

```diff
@@ -46,3 +49,21 @@ def test_print_oneline_for_invalid_yaml(self):
                 self.assertEqual(msg, 'while scanning a simple key in "file://%s", line 9, column 7 could not find expected \':\' in "file://%s", line 10, column 1' % (fullpath, fullpath))
                 print("\n", e)
                 raise
+
+    def test_print_oneline_for_errors_in_the_same_line(self):
+        # Issue #136
+        document_loader, avsc_names, schema_metadata, metaschema_loader = load_schema(
+            get_data(u"tests/test_schema/CommonWorkflowLanguage.yml"))
+
+        src = "test17.cwl"
+        with self.assertRaises(ValidationException):
+            try:
+                load_and_validate(document_loader, avsc_names,
+                                  six.text_type(get_data("tests/test_schema/"+src)), True)
+            except ValidationException as e:
+                msgs = to_one_line_messages(str(e)).split("\n")
```

Comment:
`.split(os.linesep)` or `.splitlines()` ?

Reply:
Thank you for your advice.
I replaced `str.split("\n")` with `str.splitlines()`.